packages/core/src/components/ModalNew/ModalTopActions/ModalTopActions.module.scss

```diff
@@ -2,4 +2,5 @@
   position: absolute;
   right: var(--top-actions-spacing);
   top: var(--top-actions-spacing);
+  z-index: 1;
```

Comment:
It can interrupt tipseens/dialogs that are opened on top of the modal

Reply:
Hmm... I have no choice
Images overlaps it
and basically anything that the user puts as input

Reply:
what do you suggest?

Reply:
oh I actually have an idea
```
<ModalTopActions
  renderAction={renderHeaderAction}
  color={closeButtonTheme}
  closeButtonAriaLabel={closeButtonAriaLabel}
  onClose={onClose}
/>
{children}
```

-->
```
{children}
<ModalTopActions
  renderAction={renderHeaderAction}
  color={closeButtonTheme}
  closeButtonAriaLabel={closeButtonAriaLabel}
  onClose={onClose}
/>
```

🫠

Reply:
created a task
https://monday.monday.com/boards/3532714909/pulses/7794144599